src/runmode-dpdk.c

```diff
@@ -954,6 +954,25 @@ static void DeviceSetMTU(struct rte_eth_conf *port_conf, uint16_t mtu)
 #endif
 }
 
+/**
+ * @param port_id - queried port
```

Comment:
use `\param` notation like in the rest of the code base

Reply:
changed.